parlai/agents/seq2seq/modules.py

```diff
@@ -453,7 +453,7 @@ def forward(self, input):
             scores = F.linear(e, self.weight, self.bias)
 
         if self.padding_idx >= 0:
-            scores[:, :, self.padding_idx] = -NEAR_INF
+            scores[:, self.padding_idx] = -NEAR_INF
```

Comment:
was this broken before?

Reply:
cc @stephenroller @alexholdenmiller

Reply:
seems like that. This thing is used only when lookup table is shared, so prolly was missed accidentally..

Reply:
😢  thanks for fixing!

Reply:
Are we sure about that? scores should be a (bsz x seqlen x vocab) tensor.

Reply:
Could be an issue because you limited to bsz 1? Is there an unfortunate squeeze somewhere?

Reply:
discussed offline with @stephenroller , likely happens due to bsz 1 and singleton dimension got squeezed.